project.clj

```diff
@@ -1,12 +1,13 @@
-(defproject com.cerner/clara-rules "0.22.0-SNAPSHOT"
+(defproject com.cerner/clara-rules "0.21.3-SNAPSHOT"
```

Comment:
This is an interesting version bump. Curious the rationale.

Reply:
I think at the time I did this the lint support was relatively small, but it grew, will change back.